bin/sirius-script-si-acquire-bpm-triggered-data.py

```diff
@@ -32,17 +33,17 @@ def configure_acquisition_params(orbacq):
 
 def measure(orbacq):
     """."""
-    init_state_orbacq = orbacq.get_initial_state()
+    init_state_orbacq = orbacq.get_timing_state()
     orbacq.prepare_timing()
     print('Waiting for next injection pulse')
     print('Take a look at Injection Control Log to check for next injection')
     orbacq.acquire_data()
```

Comment:
I think it is better to put this inside a `try...except` so that we always set the timing state back, even when there is some problem with the acquisition.

Reply:
Thanks for the warning. Done!